sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -628,10 +512,6 @@ def test_pred_int_tag(self, estimator_instance):
             or by defaulting to each other and/or _predict_proba
         """
         f = estimator_instance
-        # we skip the _DelegatedForecaster, since it implements delegation methods
-        #   which may look like the method is implemented, but in fact it is not
-        if isinstance(f, _DelegatedForecaster):
-            return None
 
```

Comment:
@fkiraly subclasses of `_DelegatedForecaster` were skipped before this PR. Previous discussion: https://github.com/sktime/sktime/pull/6628#discussion_r1706050435

Reply:
could we add that back? I think the skip is fine for an abstract class, but not ok for a concrete class. Sorry if it seems inconsistent, my "principle" is that properties like tags or abstract classes are ok to make distinctions about in the generic API tests, but not skips of specific classes like nbeats.

Reply:
Make sence, changes are reversed.